src/system/Progress/Progress.tsx

```diff
@@ -1,25 +1,33 @@
-/** @jsxImportSource theme-ui */
-
 /**
  * External dependencies
  */
-import React from 'react';
-import { Progress as ThemeProgress } from 'theme-ui';
-import PropTypes from 'prop-types';
+import { forwardRef, Ref } from 'react';
+import { Progress as ThemeProgress, ThemeUIStyleObject } from 'theme-ui';
 import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import { Spinner } from '../Spinner';
 import { MdCheck } from 'react-icons/md';
-import { Box, Text, Flex } from '../';
+import { Box, Text, Flex } from '..';
 
 const prefix = 'vip-progress-component';
 const uniqueID = () => Math.random().toString( 36 ).substring( 7 );
 
-const Progress = React.forwardRef(
-	( { steps, activeStep, sx, forLabel = '', className, ...props }, forwardRef ) => {
+export interface ProgressProps {
```

Comment:
I believe you can extend this from "ProgressProps" from theme-ui. It already contains sx, so you might not need to define it in the interface.

Reply:
Thanks for pointing out. I have handled this in d9e18f8bc079096d4ac847977c09bbf394dc4550